pygenometracks/plotTracks.py

```diff
@@ -150,15 +150,7 @@
 from pygenometracks._version import __version__
 from .utilities import InputError
 
-DEFAULT_BED_COLOR = '#1f78b4'
```

Comment:
why did you remove these defaults?

Reply:
Because they are not used here so I thought this would be confusing.

Reply:
Ok I see, it is because you have them set for each of the arguments that you have modified, right?

Reply:
All the default colors/colormap are defined on top of their class, the DEFAULT_TRACK_HEIGHT and DEFAULT_MARGINS are defined in tracksClass.py. This is like this since I started to work on PGT... I just decided to remove the defaults from places where they are not used... (This was done in #198 but has been merged to master not dev...).